### ISSUE TITLE
Not work Windows 11

### ISSUE THREAD

**Issue by etrigo1** [AlexxIT/HassWP#55]
Hello person, I would like some help. Does anyone know if it works on Windows 11?
I'm trying to run on Windows 11 and it doesn't get past this part. I already tested it on a friend's pc (Windows 10) and it worked.

![image](https://user-images.githubusercontent.com/37108041/175019013-50ca900d-5b20-449e-9a8a-47956e70ebdf.png)

**Comment by RobSmyth** [comment]
Yes others are seeing this too. See: https://github.com/AlexxIT/HassWP/issues/42

**Comment by EU1KY** [comment]
Just installed 2023.5.3 in Windows 11 Pro that came preinstalled on my new BeeLink Mini S tiny PC. Works fine from scratch. Winsw starts it as system service at boot (just a minor fix of mistyped Python version was needed in winsw.xml).

**Comment by AlexxIT** [comment]
Can't confirm this. I'm using Win11
